spec/request_handler/include_option_handler_spec.rb

```diff
@@ -2,26 +2,34 @@
 require 'spec_helper'
 require 'request_handler/include_option_handler'
 describe RequestHandler::IncludeOptionHandler do
+  let(:handler) do
+    options_type = Dry::Types['strict.string'].enum('user', 'email', 'user.posts')
```

Comment:
true this can be confusing - I'll think about this again and where it makes sense to use the dotted and where the double underscored names